stable/kubernetes-dashboard/templates/NOTES.txt

```diff
@@ -5,15 +5,23 @@
 {{- if .Values.ingress.enabled }}
 From outside the cluster, the server URL(s) are:
 {{- range .Values.ingress.hosts }}
+{{- if .Values.enableInsecureLogin }}
```

Comment:
$.Values , as we changed our scope here.

Reply:
Don't understand this request. You want it to say `{{- if $.Values.enableInsecureLogin }}`? If that's the request, I've never seen such syntax before.

Reply:
While you were typing that, found https://github.com/helm/helm/issues/1311 which explained it. Fixed.